charts/prometheus-rabbitmq-exporter/templates/deployment.yaml

```
@@ -77,6 +77,8 @@ spec:
               value: "{{ .Values.rabbitmq.timeout }}"
             - name: MAX_QUEUES
               value: "{{ .Values.rabbitmq.max_queues }}"
+            - name: EXCLUDE_METRICS
+              value: "{{ .Values.rabbitmq.exclude_metrics }}"
```

variables should be camelcase.
```suggestion
              value: "{{ .Values.rabbitmq.excludeMetrics }}"
```